code/datums/mutations/cluwne.dm

```diff
@@ -44,17 +35,43 @@
 				owner.Jitter(500)
 
 /datum/mutation/cluwne/on_losing(mob/living/carbon/owner)
-	owner.adjust_fire_stacks(1)
-	owner.IgniteMob()
-	owner.dna.add_mutation(CLUWNEMUT)
+	owner.emote("scream")
+	owner.visible_message("<span class='warning'><span class='name'>[owner]</span> faints as [owner.p_their()] cursed cluwne clothing melts away!</span>")
+	owner.Unconscious(rand(45 SECONDS, 70 SECONDS))
+	owner.dna.remove_mutation(CLOWNMUT)
+	owner.dna.remove_mutation(EPILEPSY)
+	for(var/datum/weakref/clothing_weakref in clothing_weakrefs)
+		var/obj/item/clothing/clothing = clothing_weakref.resolve()
+		if(QDELETED(clothing))
+			continue
+		if(!owner.doUnEquip(clothing, force = TRUE, silent = TRUE))
+			qdel(clothing)
+	clothing_weakrefs.Cut()
+
+/datum/mutation/cluwne/proc/equip_cursed_clothing(type, slot)
+	var/obj/item/clothing/original_clothing = owner.get_item_by_slot(slot)
+	if(istype(original_clothing, type))
+		return
+	if(!QDELETED(original_clothing) && !owner.doUnEquip(original_clothing, silent = TRUE))
+		qdel(original_clothing)
+	var/obj/item/clothing/cursed_clothing = new type(owner)
+	if(owner.equip_to_slot_or_del(cursed_clothing, slot))
+		clothing_weakrefs += WEAKREF(cursed_clothing)
 
-/mob/living/carbon/proc/cluwneify()
-	dna.add_mutation(CLUWNEMUT)
+/mob/living/carbon/proc/cluwneify(cursed = FALSE)
+	dna.add_mutation(cursed ? CURSEDCLUWNEMUT : CLUWNEMUT)
 	emote("scream")
 	regenerate_icons()
-	visible_message("<span class='danger'>[src]'s body glows green, the glow dissipating only to leave behind a cluwne formerly known as [src]!</span>", \
+	visible_message("<span class='danger'><span class='name'>[src]'s</span> body glows green, the glow dissipating only to leave behind a cluwne formerly known as <span class='name'>[src]</span>!</span>", \
 					"<span class='danger'>Your brain feels like it's being torn apart, there is only the honkmother now.</span>")
-	flash_act()
+	flash_act(override_blindness_check = TRUE)
+	client?.give_award(/datum/award/achievement/misc/cluwne, src)
 
-	if (client)
-		client.give_award(/datum/award/achievement/misc/cluwne, src)
+/datum/mutation/cluwne/cursed
+	mutadone_proof = FALSE // we have our own on_losing handler
+	scrambled = TRUE
+
+/datum/mutation/cluwne/cursed/on_losing(mob/living/carbon/owner)
+	owner.adjust_fire_stacks(1)
+	owner.IgniteMob()
+	owner.dna.add_mutation(CURSEDCLUWNEMUT)
```

Comment:
If you have mutadone in your system this will just activate over and over

Reply:
I think that was the point - the cursed clown mutation is admin only.

Reply:
Yeah that's correct. The cursed version is admin only, and the noncursed one is cured via applying the anti-cluwne

Reply:
Mutadone doesn't clear in a single tick. This will trigger on a loop over and over until the mutadone clears, because mutadone_proof is explicitly being disabled for some reason.

Reply:
This is the original behavior of cluwne's mutadone behavior prior to the changes